lib/generators/pakyow/app/templates/app/setup.rb

```diff
@@ -13,6 +13,7 @@
 
     # put global config here and they'll be available across environments
     app.name = '<%= app_name %>'
+    logger.enabled = true
```

Comment:
A command line option might be a good way to go. By default, the generated `Gemfile` would point at the last released version; passing an option would point it at `master`. What do you think @GuruKhalsa?